CI/org.mudlet.mudlet.yml

```diff
@@ -0,0 +1,190 @@
+app-id: org.mudlet.mudlet
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: mudlet
+# buggy - see https://github.com/flathub/org.flatpak.Builder/issues/138
+# rename-desktop-file: mudlet.desktop
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
```

Comment:
Do we *have* to use ~~pus~~*pulse*audio some people do not like to have Poetterring-werez on their machines?

Reply:
I suspect that's needed to get sound working. You don't have to use it on your machine, this is just asking flatpak to grant access to PA.

Reply:
Just to point the obvious: Flatpak depends on xdg-desktop-portal, that is needed for integration of containerized apps with host desktop session. That portal depends on Pipewire (which provides  PA via pipewire-pulse). If you're someone who objects the use of a sound server such as Pulseaudio, now replaced with Pipewire, you wouldn't be using Flatpak in the first place, so having Mudlet Flatpak use PA is objectively a non issue, and should one wish to avoid it they'll need to stick to classic packaging formats.

Reply:
*\<off-topic>Actually, my attitude to PA has softened a bit in the last few days as I have finally found out why it wasn't handling my main system's 7.1 sound card and treating it as a plain stereo device... - it is still Poettering-ware but I guess he is focussing on other things these days - given the lack of activity registered to him on their public gitlab server.\</off-topic>*